foundationdb/assets/configuration/spec.yaml

```diff
@@ -8,3 +8,34 @@ files:
   - template: instances
     options:
     - template: instances/default
+    - name: base_command
+      description: A base command to execute various queries on the FoundationDB instance
+      required: true
+      value:
+        example: fdbcli --exec
+        type: string
+    - name: cluster_file
+      description: A path to the cluster file of the FoundationDB instance. Would be added among other options to the status command
```

Comment:
The path to the cluster file of the FoundationDB instance, passed as an option to `fdbcli`. Not need if the cluster file is in the standard location (`/etc/foundationdb/fdb.cluster`).